sdk/cargo-build-sbf/src/main.rs

```diff
@@ -515,11 +515,12 @@ fn build_sbf_package(config: &Config, target_directory: &Path, package: &cargo_m
     env::set_var("OBJDUMP", llvm_bin.join("llvm-objdump"));
     env::set_var("OBJCOPY", llvm_bin.join("llvm-objcopy"));
 
+    let rustflags = env::var("RUSTFLAGS").ok();
+    let rustflags = rustflags.as_deref().unwrap_or("");
+    let rustflags = format!("{} -Zremap-cwd-prefix=", &rustflags);
```

Comment:
Do we want a cli flag to disable this behaviour?  I don't have strong opinions about this though, so happy to leave it as is until somebody has a problem with it

Reply:
I added a command line option --disable-remap-cwd to disable this remapping. By default it's enabled. Maybe we can have also an option for the user to specify what to remap to. At the moment the cwd prefix is simply removed, not replaced by anything.